docs/reference/esql/functions/description/log.asciidoc

```diff
@@ -2,4 +2,4 @@
 
 *Description*
 
-Returns the logarithm of a number to a base.
+Returns the logarithm of a value to a base. The input can be any numeric value, the return value is always a double.  Logs of zero, negative numbers, infinities and base of one return `null` as well as a warning.
```

Comment:
Nit: I don't think there's any way (or at least, there shouldn't be any non-bug way) to generate an infinite in ES|QL.  I believe we always turn them into nulls instead.